workers/onboardingEmails.js

```diff
@@ -1,11 +1,9 @@
-const moment = require('moment');
-const _ = require('lodash');
-const pushNotifications = require('../libs/pushNotifications');
-const emailsLib = require('../libs/email');
-const USERS_BATCH = 10;
+import moment from 'moment';
+import lodash from 'lodash';
```

Comment:
random, template, and assign--
if nothing else, we should probably be consistent about importing it as `_` vs `lodash`!